projects/packages/search/changelog/fix-search-options-separator-a11y

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fixed
+
+Instant Search: Hide sort options separators from screen readers.
```

Comment:
Let's consolidate these to one changelog file.

Reply:
Can do. Changes incoming...